### ISSUE TITLE
Any follow-up drafts for fully automated password change?

### ISSUE THREAD

**Issue by dumblob** [w3c/webappsec-change-password-url#34]
See https://github.com/dani-garcia/vaultwarden/discussions/1691#discussion-3362186 .

**Comment by tschoffelen** [comment]
I was reading that thread yesterday and am super interested in the concept. Happy to help further develop a specification for that, but don't know whether this is the right place, or whether it should be a separate spec.

**Comment by dumblob** [comment]
Yeah. Here I'm mainly asking for pointers to existing work on related standardizations, working groups discussing similar ideas, etc. I'm not asking for extension of this particular `webappsec-change-password-url` draft (that wouldn't make much sense IMHO).

**Comment by dumblob** [comment]
Could anyone from W3C chime in and clarify whether there is any interest or even ongoing work on the topic?

If not, I'm considering creating a GitHub org just to build some community to brainstorm some ideas and perhaps create a draft and then push the ideas to right places (maybe even RFC as I consider credentials & other secrets handling a major part of any network communication).

I know some people who have experience e.g. with the RFC processes and people from IEEE (if that's of any use, IDK). The only major element I'm missing are big commercial players which I'd very much like to incentivize to support this whole endeavor.

Anyone?

**Comment by dumblob** [comment]
ping @w3c @w3cbot @hober

**Comment by dumblob** [comment]
@oliverdunk I just saw your PR https://github.com/w3c/webappsec-change-password-url/pull/37 and thought 1Password could be interested in the automation proposed here.

Could you ping your colleagues to chime in here to get some insight how 1Password would envision such feature?

**Comment by oliverdunk** [comment]
Hey @dumblob 👋  On a personal level, I definitely think something like this would be cool! I can't speak on behalf of 1Password and I don't think anyone has thought enough about this problem to be able to weigh in on it. Do feel free to ping me again if there are any developments, though - I'd love to see them.
